alembic.ini

```diff
@@ -2,7 +2,7 @@
 
 [alembic]
 # path to migration scripts
-script_location = alembic
+script_location = bodhi:server/migrations
```

Comment:
This could be a backwards-incompatible change for the RPM, since alembic.ini is probably marked as a configuration file. This means that users who upgrade will not get this change and the migrations will fail. Two possible solutions I can think of quickly:

1. Don't mark this as a config file. Does it have anything an end user should want to change in it? I'm not sure.
2. Provide a symlink from the old location to the new location so things work smoothly, and drop the symlink in Rawhide. We already do this with another symlink on F26.

I'd probably opt for the second one.